inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -181,6 +183,7 @@ ie::CNNNetwork FrontEnd::convertNetwork(ie::CNNNetwork& network) {
     manager.register_pass<ngraph::pass::ConvertNMS3ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertNMS4ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertGather7ToGather1>();
+    // manager.register_pass<ngraph::pass::ConvertGatherND8ToGatherND5>();
```

Comment:
If it is only used by VPU please move transformation code to the plugin.

Reply:
Is it really necessary?
If so, I'll have to think a bit of how to do this.
But isn't it more convenient to store similar transformations in one place (op transformations)?

Reply:
Yes, but we do care about binary size, so we are trying to minimize common parts. So if smth is needed only by one plugin , so we move code to this plugin.

Reply:
Done. Thank you.